### PR TITLE
mkvmerge: add subtitle track example

### PR DESCRIPTION
**Pull request by CyrusYip**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/mkvmerge.md [hunk 1]
```diff
@@ -14,3 +14,7 @@
 - Extract the subtitle from track 3 of a specific file:
 
 `mkvextract tracks {{path/to/file.mkv}} {{3}}:{{path/to/subs.srt}}`
+
+- Add subtitles to a file:
+
+`mkvmerge -o {{output.mkv}} {{file.mkv}} {{subs.ass}}`
```
Comment (navarroaxel) [inline pages/common/mkvmerge.md:20]: We prefer long arguments if available.
```suggestion
`mkvmerge --output {{path/to/output.mkv}} {{path/to/file.mkv}} {{path/to/subs.ass}}`
```
Why `subs.ass` and not `.srt`?

File: pages/common/mkvmerge.md [hunk 2]
```diff
@@ -14,3 +14,7 @@
 - Extract the subtitle from track 3 of a specific file:
 
 `mkvextract tracks {{path/to/file.mkv}} {{3}}:{{path/to/subs.srt}}`
+
+- Add subtitles to a file:
```
Comment (navarroaxel) [inline pages/common/mkvmerge.md:18]: this is a track, not a video conversion
```suggestion
- Add a subtitle track to a file using:
```

File: pages/common/mkvmerge.md (hunk 1 above)
Comment (CyrusYip) [reply pages/common/mkvmerge.md:20]: The subtitle files I needed happened to be `.ass` files. It's okay to change `.ass` to `.srt`.

File: pages/common/mkvmerge.md [hunk 3]
```diff
@@ -14,3 +14,7 @@
 - Extract the subtitle from track 3 of a specific file:
 
 `mkvextract tracks {{path/to/file.mkv}} {{3}}:{{path/to/subs.srt}}`
+
+- Add a subtitle track to a file using:
```
Comment (navarroaxel) [inline pages/common/mkvmerge.md:18]: Oh, sorry, I miss the end of the example description
```suggestion
- Add a subtitle track to a file:
```

**Review by EmilySeville7cfg — APPROVED** [review]
- Approved because no unmentioned fixes were detected.
- This page should be reviewed by `mkvmerge` users.
- Thanks for a contribution. ❤️

**Review by kbdharun — APPROVED** [review]
LGTM after @navarroaxel's suggestion.

**Pull request merged**